scipy/stats/morestats.py

```diff
@@ -1568,7 +1568,8 @@ def anderson_ksamp(samples, midrank=True):
         The critical values for significance levels 25%, 10%, 5%, 2.5%, 1%.
     significance_level : float
         An approximate significance level at which the null hypothesis for the
-        provided samples can be rejected.
+        provided samples can be rejected. The value is floored / capped at
+        1% / 25%.
```

Comment:
I would add a sentence about possible future changes if or when pvalues will improved. 
So user code doesn't rely on the exact cutoff points.
IIRC, I'm using a comment in statsmodels like, users should check pvalue >=0.25 instead of pvalue == 0.25